open-api/rest-catalog-open-api.yaml

```diff
@@ -1610,13 +1610,19 @@ components:
 
     PageToken:
       description:
-        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
-        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
-        signaling to the service that the response should be paginated.
-
-        Servers that support pagination will recognize `pageToken` and return a `next-page-token` in response if there are more results available.
-        After the initial request, it is expected that the value of `next-page-token` from the last response is used in the subsequent request.
-        Servers that do not support pagination will ignore `next-page-token` and return all results.
+        An opaque token that allows clients to make use of pagination for a list APIs 
+        (e.g. ListTables). Clients may initiate the first paginated request by sending an empty 
+        query parameter `pageToken` (e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`) 
```

Comment:
I don't think there should be an "or" unless the two options are broadly considered completely equivalent. I'd prefer:
> (e.g., `GET /v1/namespaces/ns/tables?pageToken=`)

Reply:
I removed the examples.  The purpose of the OpenAPI spec is about definition valid requests, we shouldn't have to provide examples.  The two flavors are semantically the same, but some implementations can have quirks.